omniport/core/open_auth/utils.py

```diff
@@ -0,0 +1,41 @@
+from omniport.utils import switcher
+
+
+AvatarSerializer = switcher.load_serializer('kernel', 'Person', 'Avatar')
+
+def get_field_data(person, field_data_points, model_string):
+    """
+    Utility function to get requested model's data
+    :param person: person object whose data is to be retrieved
+    :param field_data_points: the specific fields of a model to be retrieved
+    :param model_string: model name string to access the data
+    :return: data for a model string
+    """
+
+    data = {}
+    if eval(model_string) is None:
```

Comment:
Is there no better way than `eval()`?

Reply:
I couldn't find any other generic work-around. And since both model strings and field data points are defined by us(no user entered string) I don't think eval would be a problem then.